src/App.tsx

```diff
@@ -23,11 +24,12 @@ const App: React.FC = () => {
       <Routes>
         <Route path='/' element={<Home />}>
           <Route path='keshav' element={<Keshav />}>
-            <Route path='week1&2' element={<Week1 />}></Route>
-            <Route path='week3' element={<Week3Main />}></Route>
-            <Route path='week4' element={<Week4 />}>
-              <Route path=':pageId' element={<Week4 />}></Route>
+            <Route path='week1&2' element={<KeshavWeek1 />}></Route>
+            <Route path='week3' element={<KeshavWeek3 />}></Route>
```

Comment:
we can use self-closing tag in these routes

Reply:
Done!